inc/saifdb.h

```diff
@@ -105,6 +107,7 @@ typedef enum _sai_fdb_entry_attr_t
     SAI_FDB_ENTRY_ATTR_META_DATA,
 
     /* -- */
+    SAI_FDB_ENTRY_ATTR_END,
 
     /* Custom range base value */
     SAI_FDB_ENTRY_ATTR_CUSTOM_RANGE_BASE  = 0x10000000
```

Comment:
IN cases where we have CUSTOME enums we should add a start and end for custom enums/user-defined enums.